client/landing/stepper/declarative-flow/internals/steps-repository/playground/lib/initialize-playground.ts

```diff
@@ -86,7 +82,7 @@ export async function initializeWordPressPlayground(
 		const client = await startPlaygroundWeb( {
 			iframe,
 			remoteUrl: 'https://playground.wordpress.net/remote.html',
-			blueprint: ! isWordPressInstalled ? getBlueprintFromUrl() : DEFAULT_BLUEPRINT,
+			blueprint: ! isWordPressInstalled ? await getBlueprintFromUrl() : DEFAULT_BLUEPRINT,
```

Comment:
If WordPress is installed, we don't even need the Blueprint.

Reply:
oh, so some meta data around what WP version and PHP version to use is stored already?

Reply:
I missed that part, we need the Blueprint to ensure the WP and PHP version.

Reply:
Should we override the WP and PHP version from the Blueprint?
Otherwise the user could use one WP/PHP version in Playground and end up with another in WP.com.

Reply:
> Should we override the WP and PHP version from the Blueprint? Otherwise the user could use one WP/PHP version in Playground and end up with another in WP.com.

I think it should be `usePhpVersions().recommendedValue`, which is the default PHP version for newly created WoA sites.

Reply:
Created a PR for handling the correct recommended php version as per WoA sites - https://github.com/Automattic/wp-calypso/pull/101826